README.md

```diff
@@ -447,6 +448,181 @@ you can create your own Configuration(s) and compose them with Config's ++ opera
+    http://openocd.org/doc/doxygen/bugs.html
+    Warn : Adapter driver 'remote_bitbang' did not declare which transports it allows; assuming legacy JTAG-only
+    Info : only one transport option; autoselect 'jtag'
+    Info : Initializing remote_bitbang driver
+    Info : Connecting to localhost:9823
+    Info : remote_bitbang driver initialized
+    Info : This adapter doesn't support configurable speed
+    Info : JTAG tap: riscv.cpu tap/device found: 0x00000001 (mfg: 0x000 (<invalid>), part: 0x0000, ver: 0x0)
+    Info : datacount=2 progbufsize=16
+    Info : Disabling abstract command reads from CSRs.
+    Info : Disabling abstract command writes to CSRs.
+    Info : [0] Found 1 triggers
+    Info : Examined RISC-V core; found 1 harts
+    Info :  hart 0: XLEN=64, 1 triggers
+    Info : Listening on port 3333 for gdb connections
+    Info : Listening on port 6666 for tcl connections
+    Info : Listening on port 4444 for telnet connections
+    
+### 5) Launch GDB
+
+Launch GDB in another terminal using
+
+    $ riscv64-unknown-elf-gdb helloworld
+    GNU gdb (GDB) 8.0.50.20170724-git
+    Copyright (C) 2017 Free Software Foundation, Inc.
+    License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>
+    This is free software: you are free to change and redistribute it.
+    There is NO WARRANTY, to the extent permitted by law.  Type "show copying"
+    and "show warranty" for details.
+    This GDB was configured as "--host=x86_64-pc-linux-gnu --target=riscv64-unknown-elf".
+    Type "show configuration" for configuration details.
+    For bug reporting instructions, please see:
+    <http://www.gnu.org/software/gdb/bugs/>.
+    Find the GDB manual and other documentation resources online at:
+    <http://www.gnu.org/software/gdb/documentation/>.
+    For help, type "help".
+    Type "apropos word" to search for commands related to "word"...
+    Reading symbols from ./proj1.out...done.
+    (gdb)
+
+Compared to Spike, the C Emulator is very slow, so several problems are encountered due to timeouts between issuing commands and response from the emulator. Thats why we should increase the `remotetimeout`.
+
+After that we load our program by performing a `load` command.
+
+ 	(gdb) set remotetimeout 2000
+ 	(gdb) target remote localhost:3333
+ 	Remote debugging using localhost:3333
+ 	0x0000000000010050 in ?? ()
+ 	(gdb) load
+ 	Loading section .text.init, size 0x2cc lma 0x80000000
+ 	Loading section .tohost, size 0x48 lma 0x80001000
+ 	Loading section .text, size 0x98c lma 0x80001048
+ 	Loading section .rodata, size 0x158 lma 0x800019d4
+ 	Loading section .rodata.str1.8, size 0x20 lma 0x80001b30
+ 	Loading section .data, size 0x22 lma 0x80001b50
+ 	Loading section .sdata, size 0x4 lma 0x80001b74
+ 	Start address 0x80000000, load size 3646
+ 	Transfer rate: 40 bytes/sec, 520 bytes/write.
+  	(gdb) 
+
+Now we can proceed as with Spike, debugging works in a similar way:
+
+	(gdb) target remote localhost:3333
```

Comment:
you don't need to target remote localhost again, you did that step above.

Reply:
Actually I'm not sure how you got from the previous step to this one. You're missing a "c" or "set breakpoint" or something.

Reply:
Yeah, I actually just copied that again, sorry. Adjusted now.